tests/lib/devicetree/api/src/main.c

```diff
@@ -17,6 +17,7 @@
 #define TEST_DEADBEEF	DT_PATH(test, gpio_deadbeef)
 #define TEST_ABCD1234	DT_PATH(test, gpio_abcd1234)
 #define TEST_ALIAS	DT_ALIAS(test_alias)
+#define TEST_HAS_ALIAS DT_HAS_ALIAS(test_alias)
```

Comment:
This macro is not as clear as simply calling `DT_HAS_ALIAS()` in the `zassert_eqaul()` IMO